zk_toolbox/README.md

```diff
@@ -7,15 +7,29 @@ Toolkit for creating and managing ZK Stack chains.
 ZK Inception facilitates the creation and management of ZK Stacks. All commands are interactive, but you can also pass
 all necessary arguments via the command line.
 
+### Dependencies
+
+Ensure you have followed
+[these instructions](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/setup-dev.md) to set up
+dependencies on your machine (don't worry about the Environment section for now).
+
+In addition to the above instructions, you will also need to
+[install Foundry](https://book.getfoundry.sh/getting-started/installation).
+
+IMPORTANT: Ensure you are running the latest stable releases of Rust using `rustup update stable` (rustc 1.78.0) & cargo
+using `cargo update` (cargo 1.78.0).
```

Comment:
just place necessary version `1.78.0` to https://github.com/matter-labs/zksync-era/blob/main/zk_toolbox/rust-toolchain

Reply:
Done 👌 Can you check zk_toolbox/rust-toolchain commit for format? I think its `stable-1.78.0' but not positive it's not just `1.78.0`